### ISSUE TITLE
fast_info broken in latest release.

### ISSUE THREAD

**Issue by fahadysf** [ranaroussi/yfinance#1636]
### Simple code that reproduces your problem

```
import yfinance as yf ; print(yf.__version__) 
# outputs 0.2.26
aapl = yf.Ticker('AAPL')
aapl.fast_info
```

### Debug log

Not much to read from debug as FastInfo is simply broken and no request is actually made.
```
>>> yf.enable_debug_mode()
>>> ticker.fast_info
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/fahad/Code/fintracker/venv/lib/python3.10/site-packages/yfinance/ticker.py", line 142, in fast_info
    return self.get_fast_info()
TypeError: 'FastInfo' object is not callable
```

### `yfinance` version

0.2.26

### Python version

Python 3.10.10

### Operating system

Mac OS 13.4.1(c) + Ubuntu 20.04.6 LTS (same behavior)

**Comment by fahadysf** [comment]
Interestingly this works
```
ticker.fast_info = yf.scrapers.quote.FastInfo(self.yfobj)
```

**Comment by Rogach** [comment]
Looks like commit 1bd819ac4da2b34995dea311268cbb2ed845c7c0 introduced the regression.

For now replacing `.fast_info` with `.get_fast_info` worked for me as a temporary workaround.

**Comment by ValueRaider** [comment]
Fixed in commit 67e81a8f9a578fa21032e1655c373b56ed65b4eb
